storage/interceptors/smaap_service_instance_interceptor.go

```diff
@@ -20,6 +20,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/Peripli/service-manager/constant"
```

Comment:
how about AroundTxUpdate - should not we skip going to the broker for reference instance updates?

Reply:
added lines 254-257
`if updatedInstance.ReferencedInstanceID != "" {
			UpdatedObject, err := f(ctx, updatedObj)
			return UpdatedObject, err
}`